src/styles/navigation_section.css

```diff
@@ -63,18 +63,19 @@
   display: none;
 }
 
-[role=banner] .coz-nav-pop--apps .coz-nav-group {
+/* coz-nav-pop--apps label is hashed coz-nav-pop--1612761714 */
+[role=banner] .coz-nav-pop--1612761714 .coz-nav-group {
```

Comment:
The hash will always be the same here? What's the interest? Just for my culture.